codegen/src/main/ops/org/nd4j/codegen/ops/Image.kt

```diff
@@ -0,0 +1,272 @@
+package org.nd4j.codegen.ops
+
+import org.nd4j.codegen.api.AtLeast
+import org.nd4j.codegen.api.Language
+import org.nd4j.codegen.api.doc.DocScope
+import org.nd4j.codegen.dsl.*
+import org.nd4j.codegen.api.DataType.*
+
+
+
+fun SDImage() =  Namespace("SDImage"){
+    val namespaceJavaPackage = "org.nd4j.linalg.api.ops.custom"
+    Op("CropAndResize") {
+        javaPackage = "org.nd4j.linalg.api.ops.impl.image"
+        javaOpClass = "CropAndResize"
+        Input(NUMERIC, "image") { description = "Input image, with shape [batch, height, width, channels]" }
+        Input(NUMERIC, "cropBoxes") { description = "Float32 crop, shape [numBoxes, 4] with values in range 0 to 1" }
+        Input(NUMERIC, "boxIndices") { description = "Indices: which image (index to dimension 0) the cropBoxes belong to. Rank 1, shape [numBoxes]" }
+        Input(NUMERIC, "cropOutSize") { description = "Output size for the images - int32, rank 1 with values [outHeight, outWidth]" }
+        Arg(NUMERIC, "extrapolationValue") { description = "Used for extrapolation, when applicable. 0.0 should be used for the default" }
```

Comment:
2 small changes here:
`Input(INT, "cropOutSize") {` - note the description mentions integer (int32) type

Let's also add a default for extrapolationValue:
`Arg(NUMERIC, "extrapolationValue") { description = "Used for ..."; defaultValue=0.0 }`